### DISCUSSION TITLE
How do I generate a new config.yml file for a new org?

### DISCUSSION THREAD

**Discussion by ryjones** [cncf/clowarden#122] [Q&A] [answered]
I'd like to [generate this file](https://github.com/cncf/people/blob/main/config.yaml) for a new org so I can start using clowarden. I looked in the docs, but I don't see a way to generate this from scratch.

I had tried using Sheriff in the past, but I could not get it running on my mac. Is there another way?

**Comment by tegioz** [comment, marked as answer]
Hi @ryjones 👋

At the moment CLOWarden does not offer a way to generate a permissions file from the current state in GitHub. IIRC that file you linked was originally generated by Sheriff actually, and given that CLOWarden is mostly compatible with that format, we kept using it once we migrated to CW. If the organization isn't too large, another option would be to prepare it manually.

CW includes a CLI tool that allows you to validate your permissions file, as well as create a diff that will detail the changes that would be applied if CW was launched. So you can use it as a guide to make sure everything is all right before deploying the CW service.

But this is something that we would like to do, so I'll create an issue to track it 🙂

Thanks!

**Reply by tegioz** [reply]
Issue created: #123
